README.md

```diff
@@ -11,9 +11,14 @@
 `git+https://github.com/QuantuMobileSoftware/sentinel2tools.git@<commit-ref>`
 <br> or
 `git+ssh://git@github.com/QuantumobileSoftware/sentinel2tools.git@<commit-ref>`
+3. If the only dependency that you didn't have installed is `Sen2Cor`, then you can use `sen2cor_install.sh` script. 
+It is added as a part of python package, so you can simply run sen2cor_install.sh - Sen2Cor will be installed in your home directory
 
 ### Usage
 
-See example.py
+See `examples/download.py` on how to download Sentinel-2 images. 
+For more details see docs in `Sentinel2Downloader.download()` method
 
-##### For more parameters see docs in download() method
+See `examples/conversion.py` on how to convert raw Sentinel-2 L1C products into L2A products. 
+In order to convert Sentinel-2 products need to be fully downloaded (.SAFE folder), 
+to achieve it use _**full_download**_ option of `Sentinel2Downloader`.
```

Comment:
Suggestion: set **full_download=True** option in `Sentinel2Downloader`.

Reply:
I don't think it is a good idea, because for the most part we don't need full .SAFE directories - we just need bands. At the moment, the only usage for this option is when we need to convert L1C product to L2A.